packages/components/datepicker/calendar-header.html

```diff
@@ -15,7 +15,7 @@
             [kbqStyle]="'transparent'"
             [tabindex]="-1"
         >
-            {{ monthSelect.triggerValue }}
+            <div class="kbq-calendar-header__select-value">{{ monthNames[selectedMonth].nameShort }}</div>
```

Comment:
Если класс нужны только для добавления 1 свойства, предлагаю использовать angular pipe

```suggestion
            {{ monthNames[selectedMonth].nameShort | titlecase }}
```